integration-tests/features/steps/gremlin.py

```diff
@@ -28,18 +29,25 @@ def gremlin_search_vertexes_for_the_ecosystem(context, ecosystem):
     post_query(context, query)
 
 
-@when('I ask Gremlin to find the package {package:S} in the ecosystem {ecosystem}')
+@when('I ask Gremlin to find all versions of the package {package:S} in the ecosystem {ecosystem}')
 def gremlin_find_package(context, package, ecosystem):
-    """Try to find the package in the selected ecosystem."""
-    query = Query().has("pecosystem", ecosystem).has("ppackage", package)
+    """Try to find all versions of the given package in the selected ecosystem."""
+    query = Query().has("pecosystem", ecosystem).has("pname", package)
```

Comment:
If you really want to make use of indexes, you can just use 
`Query().has("pname", package)` which will still give you all the version nodes given this pname

Reply:
How it works for packages with the same name, but created for different ecosystems?

Reply:
@tisnik - that's a good question. If I may ask, what is the requirement here? You need to fetch all the versions for a given ecosystem + package?

Reply:
yes

Reply:
ok. In that case, you should use something like
```
For picking only version numbers
g.V().has('ecosystem','maven').has('name','pkg-name').out('has_version').valueMap('version')

or for getting entire data for a version node
g.V().has('ecosystem','maven').has('name','pkg-name').out('has_version')
```
This query depending on the size of the data can be 10x more responsive due to proper usage of indexes. Hope it helps.

Reply:
Indeed, it helps me a lot. Thank you @miteshvp, I'll update the PR